src/player.h

```diff
@@ -223,6 +232,16 @@ namespace Player {
 	 */
 	bool IsCP1251();
 
+	/**
+	 * @return True when the DynRPG patch is active
+	 */
+	bool IsDynRpg();
```

Comment:
I know i'm being pedantic, but should these be `IsPatchDynRpg` `IsPatchManiac` etc.. Basically all of the are called `IsPatchSomething` so we know they all are detecting patches, no matter how many we support.